### PR TITLE
Updating math section

### PR DESCRIPTION
**Pull request by RajGhugare19**
Adding multi-variable calc; dividing genral math section into specific fields and adding book for maths for ml

### CODE REVIEW

**Review by GlazeDonuts — COMMENTED** [review]
Change formatting. Add heading mathematics and then  make discipline names as H2

File: mathematics/README.md [hunk 1]
```diff
@@ -19,5 +19,16 @@
...
+## Courses
+1. [Khan Academy's Multi-variable Calculus](https://www.youtube.com/watch?v=TrcCbdWwCBc&list=PLSQl0a2vh4HC5feHa6Rc5c0wbRTx56nF7)
+2. [Multi-variable Calculus by Prof. Denis Aurox](https://ocw.mit.edu/courses/mathematics/18-02-multivariable-calculus-fall-2007/)
+
+
+# Robotics
 1. [Math Fundamentals for Robotics by Michael Erdmann](http://www.cs.cmu.edu/~me/811/)
+
+# Machine Learning
+2. [Maths For Machine Learning](https://mml-book.github.io/book/mml-book.pdf)
```
Comment (akshaykulkarni07) [inline mathematics/README.md]: correct the numbering

**Review by SharathRaparthy — APPROVED** [review]
Looks good!

File: mathematics/README.md [hunk 2]
```diff
@@ -1,23 +1,36 @@
...
-# General
+
+## Multi-variable Calculus
+
+### Courses
+1. [Khan Academy's Multi-variable Calculus](https://www.youtube.com/watch?v=TrcCbdWwCBc&list=PLSQl0a2vh4HC5feHa6Rc5c0wbRTx56nF7)
+2. [Multi-variable Calculus by Prof. Denis Aurox](https://ocw.mit.edu/courses/mathematics/18-02-multivariable-calculus-fall-2007/)
+
+
+### Robotics
```
Comment (akshaykulkarni07) [inline mathematics/README.md]: do these robotics and ML sections come under multi-variable calculus?

Comment (GlazeDonuts) [inline mathematics/README.md]: I guess robotics and machine learning should be H2 instead of H3

**Review by GlazeDonuts — COMMENTED** [review]
Check the robotics and machine learning sub headings. (Suggested change to H2)

**Comment by take2rohit** [comment]
I would request @akshaykvnit to please resolve conflicts and merge to devel branch as you have a better idea of what to keep and what not to.

**Pull request merged**